platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -389,15 +388,22 @@ private MigrationReport doImport(
     } catch (MigrationException e) {
       report.record(e);
     } catch (SecurityException e) {
-      report.record(new MigrationException(Messages.IMPORT_SECURITY_ERROR, zip.getZipPath(), e));
+      report.record(
+          new MigrationException(
+              Messages.IMPORT_SECURITY_ERROR, zip == null ? exportDirectory : zip.getZipPath(), e));
```

Comment:
✏️ Should we not simply create a temp variable of type Path at the beginning with that exact logic and use that variable everywhere as opposed to duplicating it in X places? This would reduce the complexity of the method.